docs_user/assemblies/openstack_adoption.adoc

```diff
@@ -9,17 +9,11 @@ ifdef::context[:parent-context: {context}]
 :toc: left
 :toclevels: 3
```

Comment:
This comment is the for the file name. 

Do you need to add a prefix to "openstack-adoption" like "proc"?

Reply:
This assembly will eventually be removed.